internal/http/middleware/jwt/jwt.go

```diff
@@ -0,0 +1,41 @@
+package jwt
```

Comment:
What if we renamed this package to auth?

Reply:
yeah we could, It is weird to have 2 packages called jwt